### DISCUSSION TITLE
Link to "Logs" when error occurs

### DISCUSSION THREAD

**Discussion by 18923** [blakeblackshear/frigate#5551] [Ideas]
After messing up my frigate.yaml file I get a message to "Check error logs". Would it be possible to include a "link" with the error message to take you directly to the appropriate logs? Either on the same tab or open a new tab.
![Frig3](https://user-images.githubusercontent.com/59022856/220134399-559a6900-128d-4c0d-8ec6-3d9034633f1e.jpg)

**Comment by NickM-27** [comment]
The logs are right there on the left sidebar. From this screen we do not know which logs are "the important logs" as they're all combined

**Reply by 18923** [reply]
Agree, silly idea.
